lib/exd/model.ex

```diff
@@ -109,7 +109,7 @@ defmodule Exd.Model do
       defmodule unquote(module) do
```

Comment:
Why not directly move `field_attributes = Enum.flat_map(all_fields, &extract_attributes(&1))` in before?

```elixir
field_attributes = Enum.flat_map(all_fields, &extract_attributes(&1))
attribute_options = gen_attribute_options(field_attributes)
```

```elixir
if unquote(field_attributes[:name]), do: index(:name, unique: true)
```

For simplification of code and not to making things twice?